packages/editor-ui/src/components/Modal.vue

```diff
@@ -1,7 +1,7 @@
 <template>
 	<el-dialog
 		:model-value="uiStore.modalsById[name].open"
-		:before-close="closeDialog"
+		:before-close="() => closeDialog()"
```

Comment:
We should avoid defining functions in templates since they get recreated on every render. Create an extra `onCloseDialog` function instead.